arm-commerce/2015-06-01-preview/swagger/commerce.json

```diff
@@ -25,11 +25,11 @@
         "operationId": "UsageAggregates_List",
         "description": "Query aggregated Azure subscription consumption data for a date range.",
         "externalDocs": {
-          "url": "https://msdn.microsoft.com/library/azure/1ea5b323-54bb-423d-916f-190de96c6a3c"
+          "url": "https://docs.microsoft.com/rest/api/commerce/usageaggregates"
         },
         "parameters": [
           {
-            "name": "reportedstartTime",
+            "name": "reportedStartTime",
```

Comment:
The current release uses reportedStartTime, so I think this may just be a typo from when Amar created this Swagger. See https://msdn.microsoft.com/en-us/library/azure/mt219001.aspx for the current docs on this API. All of our sample code uses reportedStartTime as well. I'm not sure if the actual RP would accept either, but I wanted to correct for consistency with the samples and the other parameters (namely, reportedEndTime) to avoid confusion for customers. Commerce engineering has yet to sign off on this, but perhaps they can also weigh in.